hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileCreateRequest.java

```diff
@@ -154,10 +155,10 @@ public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
         .map(info -> info.getProtobuf(getOmRequest().getVersion()))
         .collect(Collectors.toList()));
 
+    generateRequiredEncryptionInfo(keyArgs, newKeyArgs, ozoneManager);
```

Comment:
This method's name and what it does is not consistent. I'd suggest to rename it as `setEncryptionInfo` or change it in a way that it returns a `FileEncryptionInfoProto` that is explicitly set by the client code:
```
newKeyArgs.setFileEncryptionInfo(generateEncryptionInfo(keyArgs, ozoneManager);
```